src/components/CowCard/Bloodline/Bloodline.sass

```diff
@@ -0,0 +1,38 @@
+@import ../../../styles/variables.sass
+
+.CowCard
```

Comment:
is it important to leave this wrapping selector?

Reply:
I think so! The idea is to scope these subpackage styles so that other components can potentially have a `.Bloodline` class that doesn't conflict. Admittedly this is more of a relevant concern for the new `.Subheader` class, but I figured it would be good to stick to a convention here.

Reply:
gotcha. the issue I have with it is that Bloodline is now tied to only being usable inside CowCard, but I do also understand what you're attempting to prevent from happening. this one specific css file isn't a huge deal, but maybe implementing [CSS modules](https://css-tricks.com/css-modules-part-1-need/) is worth doing at some point to have a simpler way of scoping styles in the future.

Reply:
> Bloodline is now tied to only being usable inside CowCard

Yep, that's the intention here. My thinking is that if `Bloodline` was meant to be more generally usable, it should moved to its own component directory outside of the `CowCard` directory. And in that case, we would simply remove this nesting.

> but maybe implementing [CSS modules](https://css-tricks.com/css-modules-part-1-need/) is worth doing at some point to have a simpler way of scoping styles in the future.

I would be open to that! I haven't used them at all myself, but I know that it is The New Way of organizing CSS.